### ISSUE TITLE
Pytorch 0.4.1 torch.nn.CrossEntropyLoss parameters?

### ISSUE THREAD

**Issue by LeoCHANGcn** [pytorch/pytorch#12909]
## ❓ Confused with the parameters explanation 

https://github.com/pytorch/pytorch/blob/84ce3ab47e4da0ca4727c9daf125138a7642935a/torch/nn/modules/loss.py

The class CrossEntropyLoss has a parameter `size_average=None`, but the comments above said the default value is `True`

**Comment by SsnL** [comment]
See the argument “reduction”
